src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss

```diff
@@ -0,0 +1,64 @@
+blockquote {
+  margin: 1rem 0;
+  position: relative;
+  padding: 1rem 2rem 1rem 3.5rem;
+  color: #888;
+  //font-size: .9em;
+
+  @media (min-width: $tocBreakpoint) {
+    margin: 1.5rem 2rem;
+  }
+
+  p {
+    font-size: 1.1em;
+    margin: {
+      top: 0;
+      bottom: 0;
+    }
+  }
+
+  ul, ol {
+    margin: {
+      top: .5rem;
+      bottom: .5rem;
+    }
+
+    li {
+      margin-bottom: 0 !important;
```

Comment:
I'm very bad at html/css but I was always told never to use `!important` 😬 😄

Reply:
Well, did you try to select it a bit more precisely or using the selector causing you to be forced to use an important ?

Reply:
I just copied and pasted the original css from "notice". That said, the `important` is probably justified, as there was probably another rule messing with `li` attached to a classnamed parent element, which takes precedence.

The whole theme would clearly benefit form a revamp/